api/src/main/java/cz/muni/fi/pa165/mushrooms/enums/MushroomType.java

```diff
@@ -0,0 +1,10 @@
+package cz.muni.fi.pa165.mushrooms.enums;
+
+/**
+ * Type of mushroom.
+ *
+ * @author Buvko
+ */
+public enum MushroomType {
+    EDIBLE, UNEDIBLE, POISONOUS;
+}
```

Comment:
This duplicates the enum, right? If so, we should remove it from persistence and add 'api' module to dependencies of persistence

Reply:
Are you sure we want to add this dependency?